### PR TITLE
Development branch for release v0.2 beta

### PR DESCRIPTION
**Pull request by sbreitbart-NOAA**
Attempt to inherit custom_sections parameter from asar::create_template()

Is this the correct way to inherit parameters from other functions within asar?

### CODE REVIEW

File: R/add_base_section.R [hunk 1]
```diff
@@ -1,12 +1,12 @@
 #' Add selected sections to outline
 #'
-#' @param x inherited 'custom_sections' parameter from create_template
+#' @inheritParams create_template 'custom_sections' parameter is from create_template.R
```
Comment (Schiano-NOAA) [inline R/add_base_section.R:3]: You don't need to add a description following the @inheritParams create_template. The call essentially replicates the description for any of the common parameters from.

Here is where the function gets used in create_template and it's just a way to add either all of the default sections or the ones defining custom_sections

https://github.com/nmfs-ost/asar/blob/dd56fd7c857cee1c9b9c95ed390ac5fd2785ca12/R/create_template.R#L744-L751

File: R/add_base_section.R [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 #'
 #' @return Call and copy the sections in the package templates to create an outline for a stock assessment
 #' @export
 #'
 #' @examples add_base_section(c("executive summary", "modeling approach", "results"))
-add_base_section <- function(x) {
+add_base_section <- function(custom_sections) {
```
Comment (Schiano-NOAA) [inline R/add_base_section.R:9]: this is fine

**Review by Schiano-NOAA — CHANGES_REQUESTED** [review]
Please make the change where I commented and it is good to merge into the dev branch

**Review by Schiano-NOAA — DISMISSED** [review]

**Comment by Schiano-NOAA** [comment]
@sbreitbart-NOAA what did you push with the committ merge branch 'main' into dev?

**Comment by Schiano-NOAA** [comment]
@Bai-Li-NOAA there is an issue that I can't figure out with the test-add_chunk failing. I looked over it extensively and the test *should* pass but it is not. I don't think any changes were made to that function for the test to fail. Currently, it is commented out so I can make sure the remainder of the checks are passing and I'm not going crazy lol. Would you be able to check on this by the end of next week?

**Comment by Bai-Li-NOAA** [comment]
@Schiano-NOAA, that's frustrating! Yes, I’ll check it out by the end of next week.

**Comment by Schiano-NOAA** [comment]
@sbreitbart-NOAA @Steven-Saul-NOAA @Bai-Li-NOAA as long as all goes well, I am planning on merging this dev branch for beta 3 into the main by the end of next week. Will this give everyone time to complete any remaining tasks you may want for the next addition? (We will continue to use the dev branch post merge if you don't think you can finish up stuff by next week, but this will be a large update to the main). 

Pleas reply to this comment whether this does or does not work for you and see if you can merge any current branches off the dev into dev by EOD next Friday. Thanks!!

**Comment by sbreitbart-NOAA** [comment]
> @sbreitbart-NOAA @Steven-Saul-NOAA @Bai-Li-NOAA as long as all goes well, I am planning on merging this dev branch for beta 3 into the main by the end of next week. Will this give everyone time to complete any remaining tasks you may want for the next addition? (We will continue to use the dev branch post merge if you don't think you can finish up stuff by next week, but this will be a large update to the main).
> 
> Pleas reply to this comment whether this does or does not work for you and see if you can merge any current branches off the dev into dev by EOD next Friday. Thanks!!

@Schiano-NOAA Yes, I believe that's fine with me as long as we sort out the pkgdown-vignette PR :)

**Pull request merged**